### DISCUSSION TITLE
Error in Github Documentation

### DISCUSSION THREAD

**Discussion by dynamic-modeller** [community/community#23352] [New to GitHub]
<p>hi, there is an error in the github documentation<br>
section - Importing your projects<br>
sub-section - Import Code to Github<br>
sub-sub-section - Add a Project locally</p>
<p>Step 4 you say<br>
<code>$ git init -b main</code><br>
running this command in git bash gives the error  unknown switch `b’<br>
The issue is underlined in red in the image<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/4/c4bbbfaf14921d5f3ce218ed213091406979f15a.png" data-download-href="/uploads/short-url/s4nPIdckzHBvYbZh5887kp0RGsO.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/84610991/181076474-eb052607-3fba-4ffd-a9c3-87bcbfd12544.png" alt="image" data-base62-sha1="s4nPIdckzHBvYbZh5887kp0RGsO" width="690" height="385" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4bbbfaf14921d5f3ce218ed213091406979f15a_2_690x385.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4bbbfaf14921d5f3ce218ed213091406979f15a_2_1035x577.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4bbbfaf14921d5f3ce218ed213091406979f15a_2_1380x770.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/4/c4bbbfaf14921d5f3ce218ed213091406979f15a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2151×1201 273 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>That means you’re using an old Git version and probably should update. The <code>-b</code> option <a href="https://git-scm.com/docs/git-init#Documentation/git-init.txt--bltbranch-namegt">lets you set the name of the initial branch</a>.</p>
<p>If you don’t care about the initial branch name you can remove the option and just run <code>git init</code> to use the default name, you can always rename it later.</p>

**Comment by dynamic-modeller** [comment]
<p>hey thanks for that, I had no idea, I’ll have to google how to update, thanks for the answer</p>

**Comment by tajmone** [comment]
<p>Then mark <a class="mention" href="/airtower-luna">@airtower-luna</a>’s response as the solution, so that the thread can be marked as solved.</p>

**Comment by dynamic-modeller** [comment]
<p>sorry if i am stupid, but how do i mark it as the solution, i can’t see how to do that, sorry for the dumb question</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="causalmodeller" data-post="5" data-topic="192962">
<div class="title">
<div class="quote-controls"></div>
 causalmodeller:</div>
<blockquote>
<p>how do i mark it as the solution, i can’t see how to do that,</p>
</blockquote>
</aside>
<p>Nothing to apologize for (we all start somewhere, right?), each software and platform has its own interface. See this Community post, by the admins, on how to do it and why it’s important:</p>
<p><a href="https://github.community/t/great-answer-acknowledge-it-as-a-solution/124980/4">https://github.community/t/great-answer-acknowledge-it-as-a-solution/124980/4</a></p>
<p>BTW, I advice you take the DiscoBot tutorial on how to use Discord (I took the entire tutorial at once, both beginner and advanced).  I don’t remember the exact command to start the tutorial, but you should type in the text-editing box:</p>
<p><a class="mention" href="/discobot">@discobot</a> display help</p>
<p>and the AI will show you the available options.</p>

**Comment by dynamic-modeller** [comment]
<p>hi <a class="mention" href="/tajmone">@tajmone</a> , sorry once again, but that link you gave me showed a gif of someone clicking a solution box underneath the correct answer to the problem. This solution box was immediately to the left of the heart icon</p>
<p>But i can’t see it. You will notice i have clicked heart on every reply yet, still no solution box appears. Yikes!!! Its frustrating that i cannot see a box to click. Where is this elusive solution box? I show an image of what i can see right below this line<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/9/69892b18fd796b3296990b255b7f8bd4de0fa674.png" data-download-href="/uploads/short-url/f3C10ouBKTbxG17ehCKLuQ92chC.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/84610991/181076480-cf866f0f-1bbb-4573-a140-63a660386f6b.png" alt="image" data-base62-sha1="f3C10ouBKTbxG17ehCKLuQ92chC" width="690" height="204" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/9/69892b18fd796b3296990b255b7f8bd4de0fa674_2_690x204.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/9/69892b18fd796b3296990b255b7f8bd4de0fa674_2_1035x306.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/9/69892b18fd796b3296990b255b7f8bd4de0fa674_2_1380x408.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/9/69892b18fd796b3296990b255b7f8bd4de0fa674_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1657×490 51.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="causalmodeller" data-post="7" data-topic="192962">
<div class="title">
<div class="quote-controls"></div>
 causalmodeller:</div>
<blockquote>
<p>But i can’t see it. You will notice i have clicked heart on every reply yet, still no solution box appears. Yikes!!! Its frustrating that i cannot see a box to click. Where is this elusive solution box</p>
</blockquote>
</aside>
<p>Since you’re the author of the post, you’re the only who would be seeing that button, I’m afraid. I tried to look in some of my old posts, but they either don’t have any replies I can choose as solution, or they already have one.</p>
<p>Lately I’ve been experiencing lots of issues with GitHub’s WebUI, here and on GitHub too, including missing previews of the markdown text being typed (here on Community), and missing buttons or UI elements, and very often broken CSS. Usually, refreshing the page multiple times solves the problems, but in this case I suspect it might be a backend bug which for some reason is not treating this thread (or category) as one that allows “solution answers”.</p>
<p>Well never mind. The goal of marking a solution is to allow spotting solved posts in posts-lists, and having the solution answer always show on top. But that’s not a very long thread either.</p>

**Comment by dynamic-modeller** [comment]
<p>thanks <a class="mention" href="/tajmone">@tajmone</a> , somehow when i revisted the page in response to your reply, the solution buttons were everywhere, and even better airtower-luna’s response had been selected. Thanks for helping me anyway</p>
